docs/source/install/system_requirements.rst

```diff
@@ -42,10 +42,6 @@ This is the recommended minimum sizing for testing and deploying |st2|:
 
 .. note::
```

Comment:
worth removing the _note_ block as well.
I'm guessing that's why the CI is failing